CHANGELOG.TXT

```diff
@@ -1,3 +1,18 @@
+New & Changed Features for OrangeHRM ver 4.3.5
+    Bug Fixes
+    ------------
+    Sendmail email not sending issue. Git Issue #604

```

Comment:
It's better to mention github issue number and linked

Reply:
`Git Issue #604` isn't this?